aws/eks/helm.tf

```diff
@@ -157,6 +157,13 @@ resource "helm_release" "kubernetes_external_secrets" {
   }
 }
 
+module "configmapsecrets" {
+  source                         = "github.com/mozilla-it/terraform-modules//helm/configmapsecrets?ref=master"
+  enabled                        = local.cluster_features["configmapsecrets"]
```

Comment:
I'd find it more natural to have `count = local.cluster_features["configmapsecrets"] ? 1 : 0 here. This way, we only include the module if needed, and we don't need to have the count on every resource inside the module.